azure-pipelines.yml

```diff
@@ -0,0 +1,9 @@
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: dirigeants/pipelines
+      endpoint: vladfrangu
```

Comment:
If this is from the Azure Dashboard, it should be kingdgrizzle. Also, package.json script please ❤️

Reply:
Done!